doc/release-notes/release-7.sgml.in

```diff
@@ -30,11 +30,13 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
```

Comment:
What we are describing is not really a new feature. It is more like a removal of an old feature, but there is no such section in the template AFAICT. FWIW, I am not against adding a new section in this PR if others want it (here and in template.sgml?). If others can do that instead, that would be even better because I find it difficult to work within this document organization.